core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -376,8 +376,8 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
           <div class="title-section" style="background-color: <[collectionSummary.thumbnail_bg_color]>; z-index: 1;">
             <img ng-src="<[collectionSummary.thumbnail_icon_url]>" class="thumbnail-image">
           </div>
-          <div class="title-section-mask" style="padding: 26px;">
-            <span style="font-size: 14px;">Hover over an icon to preview an exploration.</span>
+          <div class="title-section-mask objective oppia-activity-summary-tile-pre-hover-preview">
```

Comment:
Just to check, what's the aim of the .objective class? I feel like I might be missing something here.